third_party/move/testing-infra/transactional-test-runner/src/framework.rs

```diff
@@ -874,7 +951,8 @@ fn compile_ir_module<'a>(
 ) -> Result<CompiledModule> {
     use move_ir_compiler::Compiler as IRCompiler;
     let code = std::fs::read_to_string(file_name).unwrap();
-    IRCompiler::new(deps.collect()).into_compiled_module(&code)
+    let module = IRCompiler::new(deps.collect()).into_compiled_module(&code)?;
```

Comment:
A noob question, what is the difference between before and after?

Reply:
Nothing.  At some point I had added an `Option<GlobalEnv>` result to a tuple here, but then realized it wasn't useful.  I guess i didn't clean up completely.